README.md

```diff
@@ -1,2 +1,52 @@
-# testcontainers-java-module-vault
-Testcontainers module for HashiCorp Vault
+# TestContainers Vault Module
+
+Testcontainers module for [Vault](https://github.com/hashicorp/vault). Vault is a tool for managing secrets. More information on Vault [here](https://www.vaultproject.io/).
+
+See [testcontainers.org](https://www.testcontainers.org) for more information about Testcontainers.
+
+<!--[![Build Status](https://travis-ci.org/testcontainers/testcontainers-java.svg?branch=master)](https://travis-ci.org/testcontainers/testcontainers-java)-->
```

Comment:
this badge is for the main project (testcontainers-java). We should use repo-specific badge here